juniper_official/lldp/lldp-session-state.rule

```diff
@@ -23,62 +23,61 @@ healthbot {
+                    frequency 180s;
                 }
             }
             /*
              * Fields defined using sensor path. Map the longer sensor names
              * to the shorter field names used in the rules.
              */
-            field lldp-enabled {
+            field interface-name {
                 sensor lldp-sensor {
-                    path /lldp/interfaces/interface/state/enabled;
+                    where "lldp-interface-name =~ /{{interface-name}}/";
+                    path "lldp-interface-name";
                 }
                 type string;
-                description "Local interface LLDP state";
-                enumerate false as 0;
-                enumerate true as 1;
+                description "Interfaces to be monitored";
             }
-            field interface-name {
+            field lldp-neighbor-count {
                 sensor lldp-sensor {
-                    where "/lldp/interfaces/interface/@name =~ /{{interface-name}}/";
-                    path "/lldp/interfaces/interface/@name";
+                    path lldp-interface-neighbor-count;
                 }
-                type string;
-                description "Interfaces to be monitored";
+                type integer;
+                description "LLDP neighbor status";
             }
             /*
              * Anomaly detection logic.
              */
-            trigger lldp-session-state {
-                synopsis "LLDP state KPI";
+            trigger lldp-neighbor-state {
+                synopsis "LLDP neighbor state KPI";
                 description "Sets health based on lldp session state changes and notify anomaly.";
                 frequency 1offset;
                 /*
                  * Sets color to green when LLDP neighbor state is true.
                  */
-                term is-lldp-session-up {
+                term is-lldp-neighbor-discovered {
                     when {
-                        matches-with "$lldp-enabled" true;
+                        greater-than-or-equal-to "$lldp-neighbor-status" 1;
                     }
                     then {
                         status {
                             color green;
-                            message "LLDP status on interface $interface-name is Enabled";
+                            message "$lldp-neighbor-count lldp neighbors discovered on $interface-name";
                         }
                     }
                 }
                 /*
                  * Defaults color to red and notify anomaly.
                  */
-                term lldp-session-down {
+                term lldp-neighbor-undiscovered {
                     then {
                         status {
                             color red;
-                            message "LLDP status on interface $interface-name is Disabled";
+                            message "$lldp-neighbor-count lldp neighbors discovered on $interface-name";
```

Comment:
In rule properties, please add all the models that are supported by EOP. Or at least put default Junos and EVO versions. Platform ALL is not valid